cmd/shortener/main.go

```diff
@@ -29,19 +29,25 @@ func run() error {
 		return fmt.Errorf("%w", err)
 	}
 
-	s := store.NewStore(c, myLog)
-	s.ReadStoreFromFile(c)
+	s, err := store.NewStore(c, myLog)
+	if err != nil {
+		return fmt.Errorf("%w", err)
```

Comment:
При оборачивании ошибок к ним принято добавлять сообщение, поясняющее их происхождение. Типа:

```suggestion
		return fmt.Errorf("failed to initialize a new store: %w", err)
```

Reply:
готово